hack/glide-update.sh

```diff
@@ -3,4 +3,22 @@
 glide update
 
 # need to remove the vendor tree from our types to be able to build against the "right" apimachinery and such for now
-glide install --strip-vendor github.com/openshift/origin
\ No newline at end of file
+glide install --strip-vendor github.com/openshift/origin
+
+dep init
```

Comment:
what's "dep"?

Reply:
> what's "dep"?

`dep` is a golang dependency management tool which the golang guys are developing (https://github.com/golang/dep), but it doesn't allow some of the overriding control that we need (issues open).  It does have a `dep prune` command which will remove unneeded packages from the vendor folder.

`godep` allows all the control we'd need and does the pruning, but doesn't have the ability to easily update and associate with various semantic versions and branches that we need to effectively pin and vendor.  

`glide` allows the pinning we need and the semantic version tracking, but doesn't provide the pruning to create a smaller vendor folder that reflects your actual dependencies.

This pull is demonstrating how after using glide to set up a vendor folder, you can use dep to prune out the dead wood with minimal effort.

Reply:
got it, thanks!  that makes much more sense now :)